pkg/zng/zng.go

```diff
@@ -13,12 +13,21 @@ var (
 	ErrDescriptorInvalid = errors.New("zng descriptor out of range")
 	ErrBadValue          = errors.New("malformed zng value")
 	ErrBadFormat         = errors.New("malformed zng record")
-	ErrTypeMismatch      = errors.New("zng type/value mismatch")
+	ErrAccessWrongType   = errors.New("typed field access on wrong type")
```

Comment:
This doesn't seem like a better description but maybe I'm missing something.  Where are these errors cropping up?

Reply:
reverted to ErrTypeMismatch.